### ISSUE TITLE
Предлагаю переместить этот репозиторий в официальный

### ISSUE THREAD

**Issue by trikadin** [ColCh/express-snakeskin#1]
Вот, есть место, где живёт Snakeskin и всё для него. Там @kobezzza (и немножко я) развиваем его и различные адаптеры для него.
https://github.com/SnakeskinTpl

Возможно, есть смысл поговорить с кобеззой и влить этот репозиторий туда?

**Comment by ColCh** [comment]
Привет. Можно, я не против. Но для начала я бы хотел переписать биндинг с LiveScript на ES2015 :) (но это необязательно)

Зовём @kobezzza

**Comment by trikadin** [comment]
Андрей болеет, поэтому может не отвечать какое-то время.

Вообще, это необязательно. Если твоё решение работает -- это уже отлично, когда-нибудь перепишем. Например, snakeskin-loader сейчас написан на ES3 полностью (для обеспечения работы под node 0.12.x), поскольку смысла подключать бабель (который сейчас, к тому же, находится в состоянии взрыва) ради одного файла особо нет. У тебя, как я вижу, ситуация не особо другая.

Плюс, сам ты, как я понимаю, нескоро соберёшься это делать -- а там, возможно, это сделаю я или кто-то другой)

**Comment by ColCh** [comment]
Оу... 

Хорошо. Прав админа к @SnakeskinTpl у меня нет, а для трансфера это обязательное условие. У тебя есть права админа в этой организации?

**Comment by trikadin** [comment]
Да. Проблема в том, что, насколько я помню, в свой собственный проект ты меня пригласить не можешь как админа, а дать тебе права админа в @SnakeskinTpl без обсуждения с Андреем я не могу (морально). В ближайшее время я с ним это обсужу, и тогда дам тебе админ. права, и ты спокойно переместишь репозиторий туда.

**Comment by ColCh** [comment]
Окей. Ещё нужно с **npm** решить. В общем, напишите мне акки/логины, я подобавляю вас туда и удалю себя (вроде так это делается).

P.S. Если я верно понимаю, я могу передать репозиторий и права в **npm** тебе, а там уже ты можешь сам его добавить куда надо.

**Comment by trikadin** [comment]
В npm ты можешь просто добавить меня и андрея в коллабораторы:
https://www.npmjs.com/package/express-snakeskin
В правой колонке пункт collaborators, и рядом с ним плюсик.

Либо сделать `npm login`, потом `npm owner add trikadin express-snakeskin`.

**Comment by trikadin** [comment]
А перемещать его никуда не надо, зачем? Просто сделать релиз с обновлением минорной (хотя, в общем-то, можно и патч) цифры, в котором поменяешь ссылку на репозиторий в package.json

**Comment by kobezzza** [comment]
Для начала плагин нужно переписать, ибо он он использует супер древнюю версию СС. И другой вопрос: а разве СС АПИ для ноды не хватает экспрессу?

**Comment by ColCh** [comment]
@trikadin я хотел отдать вам его. получается, вы его клонируете/форкатее, обновляете в _npm_ , а здесь я в _Readme_ ставлю ссылку на репозиторий в организации – такая процедура?

@kobezzza можно пример кода по второму вопросу? Я немного не в курсе...

**Comment by trikadin** [comment]
@kobezzza В экспрессе есть вот такая штука:
http://expressjs.com/advanced/developing-template-engines.html
Ещё вот тут можно почитать:
https://github.com/tj/consolidate.js

@ColCh ты делаешь на гитхабе в настройках "Transfer ownership", выбираешь SnakeskinTpl (после того, как получишь admin rights), оно перемещается. Дальше: 
- делаешь в папке проекта `git remote set-url origin <адрес нового репозитория>` (проверь, я мог напутать в синтаксисе команды)
- В package.json удаляешь repository, bugs и homepage. 
- Делаешь `npm init` чтобы он добавил эти поля обратно, но уже для нового репозитория (это проще и надежнее, чем руками их править).
- `npm version minor`
- `git push`
- `npm publish`

Всё :smile:

**Comment by trikadin** [comment]
Ну и релиз на гитхабе сделать ещё.

**Comment by trikadin** [comment]
А все ссылки на твой старый репозиторий будут автоматически перенаправляться на новый.
